agent/runner/jobs/mongodb_restore_job.go

```diff
@@ -154,6 +154,10 @@ func (j *MongoDBRestoreJob) Run(ctx context.Context, send Send) error {
 		},
 	})
 
+	select {
+	case <-ctx.Done():
+	case <-time.After(waitForLogs):
+	}
```

Comment:
These changes are not related to the scope of the ticket. Could you describe why we need them?

Reply:
Right - they were leftovers from trying to figure out the problem, removed them now.